modules/apps/portal-vulcan/portal-vulcan-impl/src/test/java/com/liferay/portal/vulcan/internal/exportimport/data/handler/VulcanBatchEnginePortletDataHandlerTest.java

```diff
@@ -0,0 +1,163 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.vulcan.internal.exportimport.data.handler;
+
+import com.liferay.batch.engine.BatchEngineExportTaskExecutor;
+import com.liferay.batch.engine.service.BatchEngineExportTaskService;
+import com.liferay.exportimport.kernel.lar.PortletDataContext;
+import com.liferay.petra.string.StringBundler;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.test.util.RandomTestUtil;
+import com.liferay.portal.kernel.util.DateFormatFactoryUtil;
+import com.liferay.portal.test.rule.LiferayUnitTestRule;
+
+import java.io.Serializable;
+
+import java.text.DateFormat;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Map;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+import org.mockito.Mock;
+import org.mockito.Mockito;
+
+/**
+ * @author Petteri Karttunen
+ */
+public class VulcanBatchEnginePortletDataHandlerTest {
```

Comment:
I think this test is not enough. IMHO, we should create playwright tests to verify the expected behaviour based on the filter selected; otherwise we would not be testing an end-to-end process. Are you planning to do it in a different task/subtask?